signer/migrations/0003__create_tables.sql

```diff
@@ -319,3 +319,31 @@ CREATE TABLE sbtc_signer.swept_deposits (
 -- unique per 'sweep_transaction_id'.
 CREATE UNIQUE INDEX uix_swept_deposits_req_txid_req_output_index_pkgd_txid
     ON sbtc_signer.swept_deposits(deposit_request_txid, deposit_request_output_index, sweep_transaction_txid);
+
+
+CREATE TABLE sbtc_signer.bitcoin_txs_sighashes (
```

Comment:
I'd do `tx` instead of `txs` just to be consistent with the `bitcoin_tx_inputs` etc. tables.

Reply:
fixed in https://github.com/stacks-network/sbtc/commit/0703ec69542199460a6c69a92ea352d432afc624